scripts/completion/dvc.bash

```diff
@@ -1,66 +1,79 @@
 #!/usr/bin/env bash
```

Comment:
Is there a need to have `dvc.bash` in the codebase? Maybe we could create a command:
```console
$ dvc completions > /etc/bash_completion.d/dvc
```

Reply:
yeah I'm not sure how much we want to bake this into `dvc` - it's a choice between having this nearly as a stand-alone module reusable for other CLI utils or something more tightly integrated.

Reply:
also secondary point: even if we choose to bake this into `dvc` itself I would still recommend also supplying the rendered completions in the codebase as in my experience users tend to like the ability to `wget` them directly. I'd be fine with ignoring that for now and waiting for user requests though. Since most `dvc` users are some sort of devs then they might be fine with running `dvc completions | sudo tee /etc/bash_completion.d/dvc`.

Reply:
If this is autogenerated already, I don't really see a point in keeping it here. It will just take time on pre-commit hooks for no reason. Let's delete these completion files and just change the docs to use `dvc completion` command.

Reply:
removed